web/src/components/VideoPlayer.jsx

```diff
@@ -8,7 +8,7 @@ import 'videojs-seek-buttons/dist/videojs-seek-buttons.css';
 
 const defaultOptions = {
   controls: true,
-  playbackRates: [0.5, 1, 2, 4, 8],
+  playbackRates: [0.5, 1, 2, 4, 8, 16],
```

Comment:
Have we tested other browsers to ensure they also support this playback rate? For example FireFox and Safari? Would need to ensure this otherwise might need to only offer this to browsers that support it.

Reply:
I have not, but will test on those two. Any other we need to check?

Reply:
Maybe Opera and edge but I believe those are both now based on Chromium so should be same as chrome (I think)

Reply:
Cool, I think adding a check for Firefox for 8x and 16x would be best, but curious what @blakeblackshear  thinks

Reply:
8x on Firefox 99.0.1 for Mac works for me. Have never had a problem with it.

Reply:
Good to know, looks like PR only did it to 16x anyway so looks good to me